test/core/tsi/ssl_session_cache_test.cc

```diff
@@ -88,7 +88,6 @@ TEST(SslSessionCacheTest, InitialState) {
   // Verify session initial state.
   {
     tsi::SslSessionPtr tmp_sess = tracker.NewSession(1);
-    EXPECT_EQ(tmp_sess->references, 1);
```

Comment:
If tmp_sess isn't being used, does the previous line need to declare the variable at all, or can the call just be done without catching its return value?

Reply:
Asserts (`tracker.IsAlive` and `tracker.AliveCount`) verify side effects of SslSessionPtr's constructor and destructor so value has to be created and destroyed only on exiting this scope

Reply:
Right. Storing it means `SSL_SESSION_free` isn't called until the end of the function, rather than immediately.